iron-inttest/src/test/java/io/axway/iron/spi/aws/AwsKinesisTransactionStoreS3SnapshotStoreIT.java

```diff
@@ -8,30 +8,33 @@
 import io.axway.iron.spi.aws.s3.AwsS3SnapshotStoreFactory;
 import io.axway.iron.spi.jackson.JacksonSerializer;
 
-import static io.axway.iron.spi.aws.AwsProperties.S3_BUCKET_NAME_KEY;
-
+import static io.axway.iron.spi.aws.AwsProperties.*;
 
 public class AwsKinesisTransactionStoreS3SnapshotStoreIT extends BaseInttest {
 
+    private String m_directoryName;
```

Comment:
As Codacy spots, this is not needed as a member, local variable in createBucketAndStream is enough